### ISSUE TITLE
Make messages a new struct

### ISSUE THREAD

**Issue by quackduck** [quackduck/devzat#31]
Then we could have better detection if messages are dms, instead of the hacky way (as I note in the comment) we do this detection rn:
```go
if strings.HasSuffix(senderName, " <- ") || strings.HasSuffix(senderName, " -> ") { // kinda hacky
```
https://github.com/quackduck/devzat/blob/main/devchat.go#L265

**Comment by quackduck** [comment]
We could also have some other fun stuff with this feature, plus cleaner and more oop I guess

**Comment by patrickhener** [comment]
Yeah for example you could break out the dm to an own command like `./dm @someone <message>`?

**Comment by quackduck** [comment]
Actually we had that exact command but moved to = cause it’s easier to type

**Comment by quackduck** [comment]
@patrickhener so we already have that lol

**Comment by patrickhener** [comment]
fine

**Comment by quackduck** [comment]
yup
